tracer/test/Datadog.Trace.TestHelpers/PublicApiTestsBase.cs

```diff
@@ -111,10 +111,12 @@ private static bool HasHideInIntellisenseAttributes(Type type)
         private string GetExpected(string publicApi, string targetFramework = null, [CallerMemberName] string methodName = null)
         {
             // poor-man's VerifyTests.Verify, because Verify has incompatible dependencies with ASP.NET Core
+            var assemblyName = _assembly.GetName().Name;
+
             var snapshotDirectory = Path.Combine(Directory.GetParent(_filePath).FullName, "Snapshots");
             var intermediatePath = targetFramework == null ? methodName : $"{methodName}.{targetFramework}";
-            var receivedPath = Path.Combine(snapshotDirectory, $"PublicApiTests.{intermediatePath}.received.txt");
-            var verifiedPath = Path.Combine(snapshotDirectory, $"PublicApiTests.{intermediatePath}.verified.txt");
+            var receivedPath = Path.Combine(snapshotDirectory, $"PublicApiTests.{assemblyName}.{intermediatePath}.received.txt");
```

Comment:
You'll need to rename/add the 6.0 snapshots too (including the new ones for the new nuget)

Reply:
Fixed in 46ae5f8dd067fd13db2b590f9a283f7ed6c1edea